tests/core/utilities/test_abi_is_encodable.py

```diff
@@ -56,6 +56,7 @@
         ('0xname.eth', 'address', True),  # 0x in name is fine, if it is not a TLD
         ('0xff', 'address', False),  # but any valid hex starting with 0x should be rejected
     ),
+    ids=['test_' + str(x) for x in range(42)]
```

Comment:
Since these `ids` actually obscure the test results, let's just remove them now that pytest is pinned to 3.2.5.